modules/library/src/main/scala/zio/elasticsearch/executor/HttpExecutor.scala

```diff
@@ -362,6 +368,8 @@ private[elasticsearch] final class HttpExecutor private (esConfig: ElasticConfig
             e => ZIO.fail(new ElasticException(s"Exception occurred: ${e.getMessage}")),
             value => ZIO.succeed(new SearchResult(value.results.map(Item.apply)))
           )
+        case HttpUnauthorized =>
+          ZIO.fail(UnauthorizedException("Unauthorized action!"))
         case _ =>
```

Comment:
It seems it's possible to get a `403` error as well. Check [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/anonymous-access.html) for more info.